app/models/test_track/visitor.rb

```diff
@@ -98,6 +98,10 @@ def loaded?
     !offline? && @remote_visitor.present?
   end
 
+  def id_changed?
```

Comment:
Tried to follow rails naming conventions, but maybe this hides what this actually represents, the fact that the visitor id was overridden by the remote visitor?